notebooks/en/llm_gateway_pii_detection.ipynb

```diff
@@ -0,0 +1,536 @@
+{
```

Comment:
<p>might be a stupid question but I'm sure many readers will have this, can we just use something else for the name instead of hardcoding it for all possible values? like this one: <a href="https://github.com/kylemclaren/scrub" target="_blank">https://github.com/kylemclaren/scrub</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
@asusevski I think for this you can mention briefly something like "for names specifically one could use other scrubbing libraries like this one (and give link to your name scrubbing library of choice) and if you want to scrub a specific word or attribute that you have certain list of, you can do it like below" 
because name is a bit confusing example here IMO

Reply:
<p>Absolutely, and thank you for sharing this repo. It seems much better and easier to install (no poetry needed, for example). The wealthsimple repo includes a front-end and some other useful things that aren't actually necessary for this notebook. I see two options -- I can update this segment by integrating this library into the notebook for additional scrubbing or I could totally restart and solely rely on the library you shared. I'm leaning towards the first one since the wealthsimple repo also includes postgres integration and completes the wrapper for the APIs, but I'd love to hear your thoughts</p>

Reply:
<p>I've modified it to hopefully clear this up, I'd value any feedback :)</p>